src/server/field.rs

```diff
@@ -714,4 +677,56 @@ impl<M: ReadEntry, Entry> ReadEntryResult<M, Entry> {
             io::Error::new(io::ErrorKind::InvalidData, msg),
         )
     }
-}
\ No newline at end of file
+}
+
+#[derive(Debug)]
+pub enum ParseHeaderError {
+    /// The header was not found
+    Missing,
```

Comment:
see 	f429ee7 that introduces an enum to tell which header could not be found,
and 2d95162 that makes this enum and the whole `ParseHeaderError` private.